core_types/src/agent_id.rs

```diff
@@ -0,0 +1,156 @@
+//! Agent Ids are 64 bytes long.
+//! - The first 32 bytes 0-32 are the public side of a libsodium signing keypair
```

Comment:
Shouldn't it be `0-31`?

Reply:
Sorry, I tend to assume the top-end is exclusive, but I can never remember if it's a paren or bracket that denotes that. I can update to `[0-32)`.